src/main/kotlin/racing/domain/CarNameParser.kt

```diff
@@ -0,0 +1,18 @@
+package racing.domain
+
+import racing.CAR_NAME_LENGTH_LIMIT
+
+class CarNameParser {
+    fun parse(carNameLine: String): List<String> {
+        val names = carNameLine.split(CAR_NAME_DELIMITER)
+        names.forEach {
```

Comment:
[all](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/all.html)을 사용해서 다음과 같이 변경도 가능할 것 같아요!

```suggestion
        require(names.all { it.length <= CAR_NAME_LENGTH_LIMIT })
```